notifications-sample/package.json

```diff
@@ -0,0 +1,76 @@
+{
+	"name": "notifications-sample",
+	"displayName": "Notifications Sample",
+	"description": "Examples of notification types and various configurations",
+	"version": "0.0.1",
+	"engines": {
+		"vscode": "^1.69.0"
+	},
+	"categories": [
+		"Other"
+	],
+	"activationEvents": [
+		"onCommand:notifications-sample.showInfo",
+		"onCommand:notifications-sample.showInfoAsModal",
+		"onCommand:notifications-sample.showWarning",
+		"onCommand:notifications-sample.showWarningWithActions",
+		"onCommand:notifications-sample.showError",
+		"onCommand:notifications-sample.showProgress"
+	],
+	"main": "./out/extension.js",
+	"contributes": {
+		"commands": [
+			{
+				"command": "notifications-sample.showInfo",
+				"title": "Show Info Notification",
+				"category": "Notifications Sample"
+			},
+			{
+				"command": "notifications-sample.showInfoAsModal",
+				"title": "Show Info Notification As Modal",
+				"category": "Notifications Sample"
+			},
+			{
+				"command": "notifications-sample.showWarning",
+				"title": "Show Warning Notification",
+				"category": "Notifications Sample"
+			},
+			{
+				"command": "notifications-sample.showWarningWithActions",
+				"title": "Show Warning Notification With Actions",
+				"category": "Notifications Sample"
+			},
+			{
+				"command": "notifications-sample.showError",
+				"title": "Show Error Notification",
+				"category": "Notifications Sample"
+			},
+			{
+				"command": "notifications-sample.showProgress",
+				"title": "Show Progress Notification",
+				"category": "Notifications Sample"
+			}
+		]
+	},
+	"scripts": {
+		"vscode:prepublish": "npm run compile",
+		"compile": "tsc -p ./",
+		"watch": "tsc -watch -p ./",
+		"pretest": "npm run compile && npm run lint",
+		"lint": "eslint src --ext ts",
+		"test": "node ./out/test/runTest.js"
```

Comment:
If you remove `src/test` dir I'd also probably remove any testing-related npm scripts.